CHANGELOG.md

```diff
@@ -41,6 +34,8 @@ App Center Auth is [retired](https://aka.ms/MBaaS-retirement-blog-post) and has
 
 * **[Feature]** Add `UpdateTrack` property to be able to explicitly set either `Private` or `Public` update track. By default, a public distribution group is used. **Breaking change**: To allow users to access releases of private groups you now need to migrate your application to call `Distribute.UpdateTrack = UpdateTrack.Private` before the SDK start. Please read the documentation for more details.
 * **[Behavior change]** The public distribution is simplified to provide only one public group. If you have existing public groups defined for your application your users will receive the latest version of all public groups.
+* **[Feature]** Add a **Automatic Check For Update** checkbox that can be unchecked to turn off automatic check for update.
```

Comment:
```suggestion
* **[Feature]** Add an **Automatic Check For Update** checkbox that can be used to turn off automatic check for update.
```

Reply:
It's on develop and not only in Unity repo, the PR just move it to right section